test_scripts/digest_cc.sh

```diff
@@ -0,0 +1,12 @@
+TMP_SEDFILE="build/embedded_config.sed"
+
+# Digest the clienConf to hex
+EMBEDDED_CC=$(hexdump -ve '"\\\\\\\\\x" 1/1 "%.2x"' $1);
+
+# Create search and replace rule
```

Comment:
Something along the lines of "Store search-and-replace rule in a script-file" would be clearer

Reply:
Added change